src/lib/components/messages/inbox/conversationsnippet.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from "react"
+
+import {
+  MetadataText,
+  PreviewText as P,
+  SmallHeadline,
+} from "../typography"
+
+import {
+  StyleSheet,
+  ViewStyle,
+} from "react-native"
+
+import styled from "styled-components/native"
+import colors from "../../../../data/colors"
+import fonts from "../../../../data/fonts"
+import OpaqueImageView from "../../opaque_image_view"
+
+const Card = styled.View`
+    marginLeft: 20
+    marginRight: 20
+    marginTop: 20
+`
+
+const VerticalLayout = styled.View`
+    flex: 1
+    flex-direction: column
+`
+
+const HorizontalLayout = styled.View`
+    flex: 1
+    flex-direction: row
+`
+
+const CardContent = HorizontalLayout.extend`
```

Comment:
Typescript compiler complains about `extend`, but it's documented in Styled Components :o